.github/workflows/run-tests.yml

```diff
@@ -24,29 +24,19 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Cache dependencies
-        uses: actions/cache@v2
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
```

Comment:
Do you know what these two look like without a name?

I know the name generates this UI:

![image](https://user-images.githubusercontent.com/151829/127026210-9804c176-f011-46c8-937c-c96006163177.png)

Reply:
Without a name it uses "Run <action-name>", e.g. `Run actions/cache@v2`

Reply:
The actions that are just scripts show the first line of the script, so e.g. the Composer one would show `Run composer require phpunit/phpunit:${{ matrix.phpunit }} --no-update --no-interaction`